x-pack/plugin/build.gradle

```diff
@@ -18,7 +18,7 @@ dependencies {
 
 // let the yamlRestTests see the classpath of test
 GradleUtils.extendSourceSet(project, "test", "yamlRestTest", tasks.named("yamlRestTest"))
-GradleUtils.extendSourceSet(project, "test", "yamlRestCompatTest")
+GradleUtils.extendSourceSet(project, "test", "yamlRestTestV7Compat") //TODO: use version resolution
```

Comment:
@mark-vieira - Any suggestions for how read the current version so I don't need to hard code the version here ?

Reply:
Just use `VersionProperties.getElasticsearchVersion().getMajor()` to get the current major. Then I assume we just decrement it.

Reply:
perfect .. thanks!